src/main/resources/db/migration/V1_70__referral_assignment_history.sql

```diff
@@ -0,0 +1,10 @@
+create table referral_assignments
```

Comment:
the table name here is plural whilst most other tables use singular

Reply:
hmm, i think the plural makes sense to the reader, because the table is used in a collection. but i'll have a look at the existing stuff and make it as consistent as possible.

Reply:
Yeah, sorry, I think it's best to keep it as plural. It's not a straight noun as other tables are, but rather a possessive noun, so 2nd order nouns would be beneficial to have plural/singular distinction.

Reply:
👍